bookkeeper-server/src/test/java/org/apache/bookkeeper/server/http/TestHttpService.java

```diff
@@ -1087,4 +1087,28 @@ public void testSuspendCompaction() throws Exception {
         assertEquals(responseMap7.get("isMajorGcSuspended"), "false");
         assertEquals(responseMap7.get("isMinorGcSuspended"), "false");
     }
+
+    @Test
+    public void testTriggerEntryLocationCompactService() throws Exception {
+        HttpEndpointService triggerEntryLocationCompactService = bkHttpServiceProvider
+                .provideHttpEndpointService(HttpServer.ApiType.TRIGGER_ENTRY_LOCATION_COMPACT);
+
+        //1.  Put, should return OK
+        HttpServiceRequest request1 = new HttpServiceRequest(null, HttpServer.Method.PUT, null);
```

Comment:
We need to test the invalid parameter cases.

Reply:
addressed